extensions/community/OneSignal.json

```
@@ -0,0 +1,221 @@
+{
+  "author": "",
+  "category": "Third-party",
+  "extensionNamespace": "",
+  "fullName": "OneSignal",
+  "helpPath": "https://documentation.onesignal.com/docs/generate-a-google-server-api-key",
+  "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAyMy4wLjMsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjxzdmcgdmVyc2lvbj0iMS4xIiBpZD0iSWNvbnMiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4Ig0KCSB2aWV3Qm94PSIwIDAgMzIgMzIiIHN0eWxlPSJlbmFibGUtYmFja2dyb3VuZDpuZXcgMCAwIDMyIDMyOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+DQo8c3R5bGUgdHlwZT0idGV4dC9jc3MiPg0KCS5zdDB7ZmlsbDpub25lO3N0cm9rZTojMDAwMDAwO3N0cm9rZS13aWR0aDoyO3N0cm9rZS1saW5lam9pbjpyb3VuZDtzdHJva2UtbWl0ZXJsaW1pdDoxMDt9DQoJLnN0MXtmaWxsOm5vbmU7c3Ryb2tlOiMwMDAwMDA7c3Ryb2tlLXdpZHRoOjI7c3Ryb2tlLWxpbmVjYXA6cm91bmQ7c3Ryb2tlLWxpbmVqb2luOnJvdW5kO3N0cm9rZS1taXRlcmxpbWl0OjEwO30NCgkuc3Qye2ZpbGw6bm9uZTtzdHJva2U6IzAwMDAwMDtzdHJva2Utd2lkdGg6MjtzdHJva2UtbWl0ZXJsaW1pdDoxMDt9DQo8L3N0eWxlPg0KPHBhdGggY2xhc3M9InN0MSIgZD0iTTE5LDI1aDEwbC0wLjktMS4zQzI3LjQsMjIuNiwyNywyMS4zLDI3LDIwdjBjMC0xLjctMS4zLTMtMy0zaDBjLTEuNywwLTMsMS4zLTMsM3YwYzAsMS4zLTAuNCwyLjYtMS4xLDMuNw0KCUwxOSwyNXoiLz4NCjxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0yNiwyN2MwLDEuMS0wLjksMi0yLDJzLTItMC45LTItMiIvPg0KPHBhdGggY2xhc3M9InN0MCIgZD0iTTI4LDE0VjVINHYxN2MwLDAsMCw1LDAsNmwwLDBjMi43LTIuMyw1LjgtNCw5LjItNSIvPg0KPGxpbmUgY2xhc3M9InN0MSIgeDE9IjExIiB5MT0iMTIiIHgyPSIxNyIgeTI9IjEyIi8+DQo8bGluZSBjbGFzcz0ic3QxIiB4MT0iMTEiIHkxPSIxNiIgeDI9IjE0IiB5Mj0iMTYiLz4NCjwvc3ZnPg0K",
+  "name": "OneSignal",
+  "previewIconUrl": "https://asset-resources.gdevelop.io/public-resources/Icons/Line Hero Pack/Master/SVG/Communication/83c3b1f43896ce61930a58332b1fa00a1dcd9d1cabb5eec469ea54b30f173cba_Communication_chat_message_sms_bell_ring_notification.svg",
+  "shortDescription": "OneSignal is the fastest and most reliable service to send push notifications, in-app messages, SMS, and emails.",
+  "version": "1.0.0",
+  "description": "Check out the documentation here - https://documentation.onesignal.com/docs/onesignal-platform",
+  "tags": [
+    "network"
```

Let's add more tags for relevancy 💪
```suggestion
    "network",
    "mobile",
    "notification",
    "messages",
    "app",
    "OneSignal",
    "signal",
    "user",
    "retention",
    "engagement"
```